py/surveyqa/summary.py

```diff
@@ -435,36 +454,36 @@ def makeplots(exposures, tiles, outdir):
     body {
         margin: 0;
     }
-    
+
```

Comment:
All these whitespace changes are likely due to your editor being configured to auto-strip unnecessary trailing whitespace.  If you can find a way to turn that off, please do.  No need to try to undo these edits here though -- in general it is "right" to not have unnecessary trailing whitespace, but it does risk merge conflicts when your editor automatically makes updates you didn't intend.